packages/contracts-bedrock/test/L2/L1Block.t.sol

```diff
@@ -3,32 +3,53 @@ pragma solidity 0.8.15;
 
 // Testing utilities
 import { CommonTest } from "test/setup/CommonTest.sol";
+import "forge-std/console.sol";
```

Comment:
This looks like dead code, `console` shouldn't be left in the file

Reply:
removed